gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -251,6 +251,14 @@ def call_news(self, other_args: List[str]):
             default=True,
             help="Show oldest articles first",
         )
+        parser.add_argument(
+            "-s",
+            "--sources",
+            dest="n_sources",
+            default=[],
```

Comment:
So the "n_" in dest (if used) is used for number variables (int or floats).  So can we can either name this "s_sources", (s for string) or just "sources".

Reply:
I see